src/dxtbx/nexus/nxmx.py

```diff
@@ -872,7 +872,10 @@ def saturation_value(self) -> int | None:
         to the underload_value.
         """
         if "saturation_value" in self._handle:
-            return int(self._handle["saturation_value"][()])
+            try:
+                return int(self._handle["saturation_value"][()])
+            except TypeError as e:
+                logger.warning(f"Error extracting {self.path}: {e} ")
```

Comment:
In theory they could ask beamline staff about it, whether it is anything to be concerned about etc. However, because of the way we parse format classes before setting up logging in the DIALS option parser, they won't see any log messages emitted by the format class anyway...